scipy/stats/tests/test_distributions.py

```diff
@@ -2415,6 +2415,42 @@ def test_fit_duplicated_fixed_parameter(self):
         x = [0.1, 0.5, 0.6]
         assert_raises(ValueError, stats.beta.fit, x, fa=0.5, fix_a=0.5)
 
+    def test_issue_12635():
```

Comment:
I guess this is not merging with master correctly, so these are causing errors because they do not accept `self` as an argument.

Reply:
I merged master and fixed these silly errors, good catch